pkg/web/templates/components/transaction_rejection.html

```diff
@@ -31,16 +32,27 @@ <h3 class="font-bold text-xl">Reject Transaction</h3>
         </div>
         <div class="my-4">
           <label for="notes" class="label-style">Notes</label>
-          <textarea id="notes" name="notes" rows="4" class="block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-blue-500 focus:border-blue-500"></textarea>
+          <textarea id="notes" name="notes" rows="4"
+            class="block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-blue-500 focus:border-blue-500"></textarea>
         </div>
-        <div class="flex items-center gap-x-1 mb-4">
-          <label for="retry" class="ms-2 text-gray-900">Allow retry?</label>
-          <input type="checkbox" id="retry" name="retry" value="true" class="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-800 focus:ring-2">
+        <div class="flex items-center gap-x-1.5 mb-4">
+          <input type="checkbox" id="retry" name="retry" value="true"
+            class="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded focus:ring-blue-500 focus:ring-2">
+          <div class="flex items-center gap-x-1">
+            <label for="retry" class="text-gray-900">Request repair?</label>
+            <div class="tooltip tooltip-top md:tooltip-right"
+              data-tip="Select this checkbox to send a repair message to the originating VASP to try again if more information is required or something went wrong while handling the travel rule information exchange.">
```

Comment:
I slightly re-worded the text recently added to the TRISA documentation about the `Repair Workflow`.